apps/infisical-uat/scripts/copy_prod_db.sh

```
@@ -0,0 +1,200 @@
+  printf '\nThis drops every schema in %s and replaces it with a copy of %s.\nContinue? [y/N] ' "$DST_DB" "$SRC_DB"
+  read -r reply
+  case "$reply" in [yY]*) ;; *) echo "aborted"; exit 1 ;; esac
+fi
+
+# ----------------------------------------------------------------------- dump
+
+mkdir -p "$DUMP_DIR"
+
+if [ "$FROM_LATEST" -eq 1 ]; then
+  step "Selecting most recent dump"
+  DUMP="$(ls -t "$DUMP_DIR"/"$SRC_DB"-*.dump 2>/dev/null | head -1 || true)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)